src/metatrain/experimental/pet/trainer.py

```diff
@@ -676,6 +673,12 @@ def save_model(model_name, model_keeper):
             summary += f"{energies_rmse_model_keeper.best_error} "
             summary += f"at epoch {energies_rmse_model_keeper.best_epoch}\n"
 
+            if energies_mae_model_keeper.best_error < self.best_loss:
+                self.best_loss = energies_mae_model_keeper.best_error
```

Comment:
Careful here, it seems that `self.best_loss` is not a loss but something else. We should rename it because it can lead to bugs (see below)

Reply:
Eh, I kinda agree, I thought it should not be a problem because it is wrapped with if-else statement based on what we are training on

But it can break in the case we started the training on energies and then continued on forces